.jvmopts-ci

```diff
@@ -1,6 +1,13 @@
-# This is used to configure the sbt instance that github actions launches
+# This is used to configure the sbt instance in CI
 
--Xms2G
--Xmx2G
+-XX:+UseG1GC
+-Xms3G
+-Xmx3G
 -Xss2M
 -XX:ReservedCodeCacheSize=256m
+-XX:MaxGCPauseMillis=750
+-XX:-UseBiasedLocking
+-XX:+UseCompressedOops
+-XX:MetaspaceSize=512M
+-XX:-ClassUnloadingWithConcurrentMark
+-Djava.security.egd=file:/dev/./urandom
```

Comment:
Please undo this change. The `.jvmopts-ci` file is used in a lot of other jobs so changing it here will have a big impact. Set all the arguments you need directly on the `sbt ...` command.

We'll eventually converge these settings.

Reply:
discussed and agreed, I think 🙂

Reply:
Yeah, you convinced me 😅